client/my-sites/sidebar/index.jsx

```diff
@@ -1064,6 +1071,7 @@ function mapStateToProps( state ) {
 		rewindState: getRewindState( state, siteId ),
 		isCloudEligible: isJetpackCloudEligible( state, siteId ),
 		isAllSitesView: isAllDomainsView || getSelectedSiteId( state ) === null,
+		isWpMobile: isWpMobileApp(),
```

Comment:
while not wrong to pass in here this isn't needed in `mapStateToProps`. we don't expect the user-agent to change inside a browser session.


it could either be used directly in the component (which could make it harder to test if it weren't for the fact that this component is almost impossible to test already) or left here with a comment indicating it's only testing the static user agent or left here for someone else to review. maybe it's fine.

just pointing out it's qualitatively different than the other props in this function

Reply:
Agreed on all points (I just had it in the body initially them moved it so that it could be injected) – I agree, a comment would be helpful here.